buildchain/buildchain/targets/package.py

```diff
@@ -179,10 +203,25 @@ def get_source_files(self) -> types.TaskDict:
 
     def build_srpm(self) -> types.TaskDict:
         """Build the SRPM for the package."""
+        env = {
+            'SPEC': self.spec.name,
+            'SRPM': self.srpm.name,
+            'SOURCES': ' '.join(map(operator.attrgetter('name'), self.sources)),
```

Comment:
Why not use a generator expression here?

```suggestion
            'SOURCES': ' '.join(s.name for s in self.sources),
```

Reply:
Better indeed, thanks.